sdk/storage/azure-storage-blob-nio/src/main/java/com/azure/storage/blob/nio/AzureFileStore.java

```diff
@@ -22,6 +25,11 @@
     private final ClientLogger logger = new ClientLogger(AzureFileStore.class);
     private final AzureFileSystem parentFileSystem;
     private final BlobContainerClient containerClient;
+    private static final Map<Class<? extends FileAttributeView>, String> supportedAttributeViews = Map.of(
```

Comment:
`Map.of` was introduced in JDK 9, this will need to be replaced with a static initializer.